events/recorder_histogram.go

```diff
@@ -0,0 +1,71 @@
+package events
+
+import (
+	"time"
+
+	"github.com/mongodb/ftdc"
+	"github.com/mongodb/grip"
+)
+
+type histogramStream struct {
+	point     *PerformanceHDR
+	started   time.Time
+	collector ftdc.Collector
+	catcher   grip.Catcher
+}
+
+// NewHistogramRecorder collects data and stores them with a histogram
```

Comment:
Should probably mention that this is not thread safe, unless you're certain no client will try to use this from multiple threads.

Reply:
Line 26 and 27